package.json

```diff
@@ -57,6 +57,7 @@
     "vee-validate": "^3.4.5",
     "vue": "^2.6.12",
     "vue-awesome-swiper": "^4.1.1",
+    "vue-fullscreen": "^2.3.2",
```

Comment:
Needs an update to 2.3.3 to get the typings

Reply:
I saw this but the typings aren't working. Instead of importing the function to use in `vue.use`, its only importing the types

Reply:
See https://github.com/jellyfin/jellyfin-vue/pull/740/checks?check_run_id=2277616426